src/Goteo/Library/Forms/Model/ProjectCampaignForm.php

```diff
@@ -23,25 +23,21 @@
 use Goteo\Library\Text;
 use Goteo\Model\User;
 
-class ProjectCampaignForm extends AbstractFormProcessor implements FormProcessorInterface {
+class ProjectCampaignForm extends AbstractFormProcessor implements FormProcessorInterface
+{
 
-    public function getConstraints($field): array
+    public function getPaypalConstraints(): array
     {
-        $constraints = [];
-
-        if($field === 'paypal') {
-            $constraints[] = new Constraints\Email([
-                'checkMX' => true,
-                'checkHost' => true
-            ]);
```

Comment:
SF 4.2 => `checkMX` and `checkHosts` options were removed in `Constraints\Email` class